.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: CI
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```

Comment:
Any reason you're doing this?

Reply:
Mostly carried into this from other examples. It should silence the annoying "outdated pip" warning, but I agree that doesn't really matter in CI.